shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -877,13 +911,47 @@ SemanticsObject hitTest(float[] point) {
         boolean isFocusable() {
             int scrollableActions = Action.SCROLL_RIGHT.value | Action.SCROLL_LEFT.value
                     | Action.SCROLL_UP.value | Action.SCROLL_DOWN.value;
+            if (hasFlag(Flag.IS_ROUTE)) {
+                return false;
+            }
             return (actions & ~scrollableActions) != 0
                 || flags != 0
                 || (label != null && !label.isEmpty())
                 || (value != null && !value.isEmpty())
                 || (hint != null && !hint.isEmpty());
         }
 
+        void collectRoutes(List<SemanticsObject> edges) {
+            if (hasFlag(Flag.IS_ROUTE)) {
+                edges.add(this);
+            }
+            if (children != null) {
+                for (int i = 0; i < children.size(); ++i) {
+                    children.get(i).collectRoutes(edges);
+                }
+            }
+        }
+
+        String getRouteName() {
+            // Returns the first non-null and non-empty semantic value of a child
+            // with an isRouteName flag. Otherwise returns null.
+            if (hasFlag(Flag.IS_ROUTE_NAME)) {
+                String routeName = getValueLabelHint();
```

Comment:
On iOS we are just using the accessibilityValue, why use all three on Android?

Reply:
See below - we should probably just use label for both it sounds like